lisa/sut_orchestrator/azure/features.py

```diff
@@ -203,41 +195,135 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
         self._initialize_information(self._node)
 
 
-class DiskEphemeral(AzureFeatureMixin, features.DiskEphemeral):
-    def _initialize(self, *args: Any, **kwargs: Any) -> None:
-        super()._initialize(*args, **kwargs)
-        self._initialize_information(self._node)
+# disk types are ordered by cost. The earlier is lower cost.
+_ordered_disk_types: List[schema.DiskType] = [
+    schema.DiskType.StandardHDDLRS,
+    schema.DiskType.StandardSSDLRS,
+    schema.DiskType.Ephemeral,
+    schema.DiskType.PremiumLRS,
```

Comment:
Shouldn't `Ephemeral` be the least cost disk? And we use StandardHDD as default because it is most common disk option.